experiment/analysis/comparison-of-bfs-with-dfs.md

```diff
@@ -1,39 +1,43 @@
 ### Differences between BFS and DFS
 
-**Key Differences Between BFS and DFS**
+**Key differences between BFS and DFS**
 
-  -  BFS is vertex-based algorithm while DFS is an edge-based algorithm.
-  - Queue data structure is used in BFS. On the other hand, DFS uses stack or recursion.
+  -  BFS is a vertex-based algorithm while DFS is an edge-based algorithm.
+  -  Queue data structure is used in BFS. On the other hand, DFS uses stack or recursion.
   -  Memory space is efficiently utilized in DFS while space utilization in BFS is not effective.
   -  BFS is an optimal algorithm while DFS is not optimal.
-  -  DFS constructs narrow and long trees whereas BFS constructs wide and short tree.
+  -  DFS constructs narrow and long trees whereas BFS constructs wide and short trees.
 
-### BFS traversal on Example graph
-  -  We have a graph whose vertices are A, B, C, D, E, F, G. Considering A as starting point, the steps involved in the process are:
-  - Vertex A is expanded and stored in the queue.
-  -  Vertices B, D and G successors of A, are expanded and stored in the queue. Meanwhile Vertex A is removed.
+### BFS traversal on example graph
+  -  We have a graph whose vertices are A, B, C, D, E, F, G. Considering A as the starting point, the steps involved in the process are:
+  -  Vertex A is expanded and stored in the queue.
+  -  Vertices B, D, and G, as successors of A, are expanded and stored in the queue. Meanwhile, Vertex A is removed.
   -  Now B at the front end of the queue is removed along with storing its successor vertices E and F.
-  -  Vertex D at the front end of the queue is removed, and its connected node F is already visited.
-  -  Vertex G is removed from the queue, and it has successor E which is already visited.
+  -  Vertex D at the front end of the queue is removed, and its connected node F has already been visited.
+  -  Vertex G is removed from the queue, and it has successor E which has already been visited.
   -  Now E and F are removed from the queue, and its successor vertex C is traversed and stored in the queue.
-  -  At last C is also removed and the queue is empty which means we are done.
+  -  At last C is also removed and the queue is now empty which means we are done.
   -  The generated Output is – A, B, D, G, E, F, C.
 
-### DFS traversal on Example graph
+  <img src="images/bfs.png"/>
 
-Similar to BFS lets take the same graph for performing DFS operations, and the involved steps are:
+### DFS traversal on example graph
+
+Similar to BFS lets take the same graph for performing DFS operations. The steps involved in the process are:
```

Comment:
Should it be:
Similar to BFS, let's take the same graph for performing DFS operations

Reply:
Yeah, lets is supposed to be a contraction, and the comma adds clarity.